agent/src/main/java/com/code_intelligence/jazzer/agent/RuntimeInstrumentor.kt

```diff
@@ -57,10 +57,20 @@ internal class RuntimeInstrumentor(
             }
         }
         .flatMap { loadHooks(it) }
-    private val customHooks = emptyList<Hook>().toMutableList()
+
```

Comment:
How about we add a FIXME comment here: With the current logic, every time we add a class to the new list because it was mentioned by a hook, _every_ custom hook will be applied to it. As we add more hooks, that may have undesired consequences, so at some point we might want to track the list of classes per custom hook rather than globally.